### ISSUE TITLE
Inkscape 1.3 reports Unknown option --pdf-page=1

### ISSUE THREAD

**Issue by Zane-XY** [textext/textext#386]
Trying to install this on Inkscape 1.3 and it does not work anymore. Inkscape 1.2 has many bugs on Mac m2 chips, so I have to use 1.3beta instead.

Inkscape 1.3-beta (cedbd6c, 2023-05-28)

    GLib version:     2.72.1
    GTK version:      3.24.37
    glibmm version:   2.60.0
    gtkmm version:    3.24.5
    libxml2 version:  2.10.3
    libxslt version:  1.1.35
    Cairo version:    1.17.9
    Pango version:    1.50.6
    HarfBuzz version: 2.9.1

    OS version:       macOS 13.2.1 (22D68) arm64

Installed with `textext-1.8.2`, and click the preview or save, it gives the following error:
```
Command /Applications/Inkscape.app/Contents/MacOS/inkscape --pdf-poppler --pdf-page=1 --export-type=png --export-area-drawing --export-dpi=300 --export-filename tmp.png tmp.pdf failed (code 1)

Unknown option --pdf-page=1
```

<img width="450" alt="image" src="https://github.com/textext/textext/assets/2939476/5d0dc273-3797-4cfb-9c33-00a10bfbf7ff">

**Comment by jcwinkler** [comment]
I confirm this issue. The option has been removed. Working on a fix.

**Comment by jcwinkler** [comment]
@Zane-XY You find attached a version with a fix. Please install and try it. Does it work on MacOS?

[TexText-Macos-1.8.3-dev.zip](https://github.com/textext/textext/files/11674908/TexText-Macos-1.8.3-dev.zip)

**Comment by Zane-XY** [comment]
@jcwinkler 
Hi, the new version works. 
Thanks!

**Comment by quctex** [comment]
You may try the workaround described in #390 to work with TexText in Inkscape 1.3 now.

**Comment by jcwinkler** [comment]
Fixed in release 1.9.0.
